contract/views/contract.xml

```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!--Main FORM view-->
+    <record id="contract_contract_form_view" model="ir.ui.view">
+        <field name="name">contract.contract form view (in contract)</field>
+        <field name="model">contract.contract</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button name="action_contract_send"
+                            type="object"
+                            string="Send by Email"
+                            groups="base.group_user"/>
+                    <button name="recurring_create_invoice"
+                            type="object"
+                            attrs="{'invisible': [('create_invoice_visibility', '=', False)]}"
+                            string="Create invoices"
+                            groups="base.group_no_one"/>
+                </header>
+                <sheet string="Contract">
+                    <div class="oe_button_box" name="button_box">
+                        <button class="oe_stat_button" type="object"
```

Comment:
Instead of this, can you put `<widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>` just after sheet? Currently the archiving does not look good:

![image](https://user-images.githubusercontent.com/19620251/80123016-9f23df80-858e-11ea-886f-74b0a7bc4cea.png)

Reply:
done